ext_tables.php

```diff
@@ -92,6 +92,9 @@
     \TYPO3\CMS\Core\Utility\ExtensionManagementUtility::allowTableOnStandardPages('tx_styleguide_inline_1n');
     \TYPO3\CMS\Core\Utility\ExtensionManagementUtility::allowTableOnStandardPages('tx_styleguide_inline_1n_child');
 
+    \TYPO3\CMS\Core\Utility\ExtensionManagementUtility::allowTableOnStandardPages('tx_styleguide_inline_1_1');
```

Comment:
i would prefer to get the naming right - the table names are a convention for the data generator to find "main" tables, see getListOfStyleguideMainTables() in class Generator.